React/CoreModules/RCTDevSettings.h

```diff
@@ -48,16 +48,6 @@
 
 @end
 
-@protocol RCTDevSettingsInspectable <NSObject>
```

Comment:
0.66 explicitly added this. What's the build error? I'd imagine if anything we need to add something not remove this.

Reply:
I think it was the this code was duplicated during merge. I see it repeat higher in the file.